Makefile

```diff
@@ -59,8 +59,9 @@ ec:
 libsodium:
 ifeq (,$(wildcard ${STATIC_LIB}.*))
 	@echo Static libsodium was not found at ${STATIC_LIB} so compiling libsodium from source.
-	@cd $(LIBSODIUM_DIR)/ && ./configure  \
-		--enable-static --enable-shared --with-pic --prefix="$(INSTALL_DIR)"
+	@cd $(LIBSODIUM_DIR)/ && ./autogen.sh
```

Comment:
I added autogen.sh because it generates Makefile and configure for libsodium. I suppose the libsodium configure you have in your repo should be generated by autotools.

(I accidentally posted this comment to the pull request itself)